src/markdown.c

```diff
@@ -1403,7 +1403,7 @@ prefix_quote(uint8_t *data, size_t size)
 	if (i < size && data[i] == ' ') i++;
 	if (i < size && data[i] == ' ') i++;
 
-	if (i < size && data[i] == '>') {
+	if ((i < size && data[i] == '>') && (i + 1 < size && data[i+1] != '!')) {
```

Comment:
I think this changes the behavior of > to not detect an empty block quote at the end of the text. Not necessarily a concern though.

">", size=1, i=0, i+1 =1 which is not < size
so the if expression evaluates to: `true && true && false && unevaluated` when it used to be `true && true`